docs/Lifecycle.md

```diff
@@ -25,8 +25,10 @@ Incoming Request
                                      4**/5** ◀─┴─▶ User Handler
                                                      │
                                                      └─▶ Reply
-                                                          │ │
-                                                          │ └─▶ Outgoing Response
-                                                          │
-                                                          └─▶ onResponse Hook
+                                                           │
+                                                           └─▶ onSend Hook
+                                                                  │
+                                                        4**/5** ◀─┴─▶ Outgoing Response
+                                                                        │
+                                                                        └─▶ onResponse Hook
```

Comment:
In this graph the serialization phase is not described. Could you add it?
cc @delvedor

Reply:
It is not described because is inside `Reply` and cannot cause errors :)

Reply:
Yeah but `Reply` is  not a call step: it is an object/class. And `Reply` is the only non-step here.
Can we replace `Reply` with `serialization`?

Reply:
`Reply`is not just the serialization, but also header and status code (but also more, you can extend it!), so I think it is correct to leave just `Reply` there, because contains al the stuff you can do with it.
If you don't like it, we should write something like
```
└─▶ serialization
    headers
    status codes
    extended utilities
       └─▶
```
But is not that good :P